Scripts/gen_Primary.sh

```diff
@@ -3,13 +3,14 @@
 # create fcl for producing primaries from stopped particles
 # this script requires mu2etools and dhtools be setup
 #
-# Usage: ./Production/Scripts/generate_Primary.sh --primary CeEndpoint --campaign MDC2020 --pver v --sver p --type Muminus --njobs 1000 --events 4000 --pdg 11 --start 0 --end 110 --field Offline/Mu2eG4/geom/bfgeom_reco_altDS11_helical_v01.txt
+# Usage: ./Production/Scripts/generate_Primary.sh --primary CeEndpoint --PCAMPAIGN MDC2020 --pver v --sver p --type Muminus --njobs 1000 --events 4000 --pdg 11 --start 0 --end 110 --field Offline/Mu2eG4/geom/bfgeom_reco_altDS11_helical_v01.txt
 #
 # Note: User can omit flat (pdg, startmom and enedmom) arguments without issue. Field argument also generally will not be used
 
-# The main input parameters needed for any campaign
+# The main input parameters needed for any PCAMPAIGN
 PRIMARY="" # is the primary
-CAMPAIGN="" # Campaign (MDC2020"
+PCAMPAIGN="" # primary CAMPAIGN (MDC2020"
```

Comment:
please revert the name change (pcampaign->campaign) and make scampaign optional with default value=campaign. That maintains backwards-compatibility and simplifies the interface.

Reply:
done